widgetssdk/src/main/java/com/glia/widgets/di/Dependencies.java

```diff
@@ -214,6 +212,7 @@ public static void setControllerFactory(ControllerFactory controllerFactory) {
     }
 
     public static Navigator getNavigator(@Nullable ArrayList<String> queues, @Nullable String contextAssetId) {
+        Navigator navigator = new Navigator();
         navigator.setQueueIds(queues);
         navigator.setContextAssetId(contextAssetId);
```

Comment:
Could you please change these setters with constructor parameters?

Reply:
Good point, changed 5007a98c245dc7ef1c7906c944cd16bcaf726a1c